core/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM python:3.6-alpine
 MAINTAINER Abhilash Raj
 
 #Add startup script to container
-ADD assets/run.sh /opt/run.sh
+COPY docker-entrypoint.sh /usr/local/bin/
```

Comment:
I highly discourage that, as it's unnecessary verbosity. I have seen no Dockerfile (including official ones) chmod'ing scripts when copying. (Of course, for downloaded files, that's necessary.)